src/content.ts

```diff
@@ -801,17 +801,14 @@ function getYouTubeVideoID(url: string) {
     }
 
     //Get ID from searchParam
-    if (urlObject.searchParams.has("v") && ["/watch", "/watch/"].includes(urlObject.pathname) || urlObject.pathname.startsWith("/tv/watch")) {
+    let m = null;
+    if (urlObject.searchParams.has("v") && urlObject.pathname.match(/^\/((youtube\.com\/)?watch\/?$|tv\/watch)/)) {
         let id = urlObject.searchParams.get("v");
         return id.length == 11 ? id : false;
-    } else if (urlObject.pathname.startsWith("/embed/")) {
-        try {
-            return urlObject.pathname.substr(7, 11);
-        } catch (e) {
-            console.error("[SB] Video ID not valid for " + url);
-            return false;
-        }
-    } 
+    }
+    else if (m = urlObject.pathname.match(/^\/(?:youtube.com\/)?(?:embed|youtu.be)\/([^/]{11})/)) {
```

Comment:
What is `youtu.be` for?

Reply:
see above

Reply:
`youtu.be` should never be a valid URL since it is always redirected

Reply:
I just tested it, and youtube-local does not redirect the `/youtu.be/<video_id>` path to `/youtube.com/watch?v=video_id`.
It just serves the video as if `/youtube.com/watch?v=video_id` was used.
Two ways:
  * Merge it as is. The allowance of `/youtu.be/<video_id>` is not "hurting" anoyne, right?
  * Remove `youtu.be` from the regex, and open a ticket on youtube-local. I believe that it should be a quick fix, but I am not 100% sure.. Maybe not being redirected is a feature (idk)?